### PR TITLE
Update and rename resurrection_strings.xml to resurrection_strings-hi.xm...

### PR DESCRIPTION
**Pull request by the-dagger**
hindi translation

### CODE REVIEW

**Comment by xenon92** [comment]
If I merge this pull request, our english version of resurrection_strings.xml would get deleted. See the "Files Changed" tab.

Either create a new pull request or add another commit to this pull request and move the file to "res/values-hi/"

So the final path would be "res/values-hi/resurrection_strings-hi.xml"

Recheck the "Files Changed" tab.

**Comment by westcripp** [comment]
thanks translation has been added to settings

File: res/values/resurrection_strings-hi.xml [hunk 1]
```diff
@@ -0,0 +1,741 @@
...
+    <string name="enable_subcat_title">स्टाइलस</string>
+    <string name="enable_spen_title_head">लेखनी इशारों को सक्षम करें</string>
+    <string name="enable_spen_summary_head">लेखनी बटन के कड़ी चोट इशारों से कस्टम क्रिया लॉन्च. चेतावनी: Apps लेखनी बटन पकड़े द्वारा चलाई निवेश प्राप्त करने में सक्षम नहीं होगा</string>
+    <string name="gestures_subcat_title">इशारा</string>
+    <string name="gestures_left_spen_title_head">Left स्वाइप</string>
+    <string name="gestures_left_spen_summary_head">Left स्वाइप पर कस्टम कार्रवाई शुरू</string>
+    <string name="gestures_right_spen_title_head">Right स्वाइप</string>
+    <string name="gestures_right_spen_summary_head">Right स्वाइप पर कस्टम कार्रवाई</string>
+    <string name="gestures_up_spen_title_head">Up स्वाइप</string>
+    <string name="gestures_up_spen_summary_head"Uup swipe पर कस्टम कार्रवाई</string>
```
Comment (xenon92) [inline res/values/resurrection_strings-hi.xml:145]: Typo

Has been fixed and translation has been added. Thanx for the translation.

Comment (the-dagger) [inline res/values/resurrection_strings-hi.xml:145]: My bad..

File: res/values/resurrection_strings-hi.xml [hunk 2]
```diff
@@ -0,0 +1,741 @@
...
+    <!-- Sound settings screen, setting check box label -->
+    <string name="camera_sounds_title">कैमरा शटर ध्वनि</string>
+    <string name="camera_sounds_summary">चेतावनी: कैमरा शटर ध्वनि को अक्षम करना कुछ क्षेत्रों में अवैध है!</string>
+    <!-- Stylus Icon -->
+    <string name="stylus_icon_enabled_title">लेखनी का उपयोग करते समय आइकन दिखाएं</string>
+    <string name="stylus_icon_enabled_summary">लेखनी का उपयोग करते समय सूचक चिह्न दिखाएँ</string>
+
+    <!-- Stylus Gestures -->
+    <string name="gestures_settings_title">स्टाइलस का इशारा</string>
+    <string name="category_spen_title">Sस्टाइलस का इशारा</string>
```
Comment (the-dagger) [inline res/values/resurrection_strings-hi.xml:135]: Here is one more typo..
I added the "s" by mistake.

**Comment by westcripp** [comment]
whats the line number?

**Comment by the-dagger** [comment]
135..
Sorry I forgot to mention.

**Pull request closed without merging**